.github/workflows/java11.yml

```diff
@@ -0,0 +1,33 @@
+name: Java CI 11
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: maven
```

Comment:
FYI: instead of having a separate workflow for each java version you could define a matrix with the different version. 

For example;

```yml
jobs:
  build:
    strategy:
      matrix:
        java_version: [8, 11, 17]

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3

    - name: Set up JDK ${{ matrix.java_version }}
      uses: actions/setup-java@v3
      with:
        java-version: ${{ matrix.java_version }}
        distribution: 'temurin'
        cache: maven
     # ...
```

See for details https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs